deadpool.py

```diff
@@ -369,14 +369,36 @@ def done_with_process(self, wp: WorkerProcess):
 
     def run_task(self, fn, args, kwargs, timeout, fut: Future):
         try:
-            worker: WorkerProcess = self.get_process()
-            try:
-                worker.submit_job((fn, args, kwargs, timeout))
-            except (pickle.PicklingError, AttributeError) as e:
-                # If the user passed in a function or params that can't
-                # be pickled, use the future to communicate the error.
-                fut.set_exception(e)
-                self.done_with_process(worker)
+            retry_count = 10
```

Comment:
The retry here is only to prevent infinite loops gumming up the works. It really shouldn't happen that multiple processes are left after the previous job in a state where they'll immediately raise `BrokenPipeError`.